superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -481,6 +497,27 @@ export default class ResultSet extends React.PureComponent<
     return <div className="noControls" />;
   }
 
+  rowsReturned() {
+    const { query } = this.props;
+    const limitWarning = <Icon className="ReturnedRowsImage" name="warning" />;
+    return (
+      <ReturnedRows>
+        {query.results?.displayLimitReached && limitWarning}
+        <span>{t(`%s rows returned`, query.rows)}</span>
+        {query.results?.displayLimitReached && (
+          <span className="LimitMessage">
+            {t(
+              `It appears that the number of rows in the query results displayed
+           was limited on the server side to
+           the %s limit.`,
+              query.rows,
```

Comment:
this is cool. I didn't know that you can do string interpolation in the translate function.